### PR TITLE
UG: add edit commands

### PR DESCRIPTION
**Pull request by Thing1Thing2**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -210,6 +210,23 @@ Format: `delClient i/INDEX`
...
 * `delClient i/2`
 
+#### 3.1.5 Editing a client: `editClient`
+
+Edits detail of the specified client.
+
+Format: `editClient i/INDEX [n/NAME] [p/PHONE_NUMBER] [a/ADDRESS] [e/EMAIL] [b/BIRTHDAY] [pd/PRODUCT]`
+
+* Edits the client at the specified `INDEX`.
+  *`INDEX` refers to the index number shown by executing [`listClient`](#312-listing-all-clients--listclient) command.
```
Comment (rexong) [inline docs/UserGuide.md:220]: I think the indentation here is off

File: docs/UserGuide.md [hunk 2]
```diff
@@ -266,6 +283,23 @@ Format: `delMeeting i/INDEX`
...
+* Edits information about the meeting at the specific `INDEX`.
+* The index refers to the index number shown in the displayed meetings' list.
+* The index **must be a positive integer** 1, 2, 3, …​
+* At least one optional detail must be modified.
+* Maintains values of details not edited by the command.
+
+Examples:
+Suppose MyInsuRec contains only one meeting as created in the [`addMeeting`](#adding-a-meeting--addmeeting) command:
+* `editMeeting i/1 dn/Follow up team meeting` changes the description of this meeting.
+* `editMeeting i/1 st/1500 et/1200` will show error saying invalid time since start time is later that end time.
```
Comment (sikai00) [inline docs/UserGuide.md:301]: typo in that

**Review by sikai00 — APPROVED** [review]

File: docs/UserGuide.md [hunk 3]
```diff
@@ -266,6 +283,23 @@ Format: `delMeeting i/INDEX`
...
 * `delMeeting i/2`
 
+#### 3.2.5 Editing a meeting: `editMeeting`
+
+Edits details of the specified meeting.
+
+Format: `editMeeting i/INDEX [d/DATE] [st/START TIME] [et/END TIME] [dn/DESCRIPTION]`
+
+* Edits information about the meeting at the specific `INDEX`.
+* The index refers to the index number shown in the displayed meetings' list.
```
Comment (rexong) [inline docs/UserGuide.md:293]: i think you should keep it consistent with what you have done for `editClient` above. ie `index` => `INDEX` and link the meeting list

**Review by rexong — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (Thing1Thing2) [reply docs/UserGuide.md:220]: ahh ok

**Pull request merged**